rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/PolicyConfigResource.java

```diff
@@ -62,7 +62,14 @@
     @Override
     public Map<String, Object> batchConfigRead(String application, String entityToken, String policyToken) {
         // TODO: add test
+        Entity entity = brooklyn().getEntity(application, entityToken);
+        if (entity != null && !Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.SEE_ENTITY, entity)) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to see the entity '%s'",
```

Comment:
Message should say `User '%s' is not authorized to see the configuration for the policy '%s'`
Please also check the parameters for the templated message